airflow/www/static/js/dag/details/taskInstance/ExtraLinks.tsx

```diff
@@ -53,6 +53,14 @@ const ExtraLinks = ({
   const isExternal = (url: string | null) =>
     url && /^(?:[a-z]+:)?\/\//.test(url);
 
+  const isSanitised = (url: string | null) => {
+    if (!url) {
+      return true;
```

Comment:
Should it be false instead?
```suggestion
      return false;
```

Reply:
My rationale here was that an empty url is a safe url

Reply:
ah, makes sense

Reply:
but after a second thought, do we need to enable it in that case?

Reply:
I retained the existing behaviour. I am open to the other way around too

Reply:
Sounds reasonable. not a blocker from me. I'll approve it

Reply:
yeah. It's fine.